js/InitElement.js

```diff
@@ -0,0 +1,142 @@
+import {
+    getUserIdAndDeleteTodolist,
+    getUserIdAndItemComplete,
+    getUserIdAndModifyTodolist,
+    getUserIdAndAddTodolist
+} from './Server.js';
+
+let $count = 0 ;     //할 일 갯수 세는 용
+function initElement(value, item, cause_of_add) {
+    const $todo_list = document.querySelector('.todo-list');
+    const content = (value)=>`
+        <li class="active"> 
+            <div class="view">
+                <input class="toggle" type="checkbox">
+                <label class="label">${value}</label>            
+                <button class="destroy"></button>    
+            </div>
+        </li>
+    `
+    $todo_list.innerHTML += content(value);
+    //localStorage의 value(item)이 T이면 completed한 상태 및 체크상태 유지(아닐경우 기본 active와 체크상태 X이다)
+
+    if(item === 'T')
+    {
+        document.querySelectorAll('.label').forEach($el => {
+            if($el.innerText === value){
+                $el.closest('li').setAttribute('class' , 'completed');
+                $el.closest('div').children[0].setAttribute('checked','true');
+            }
+
+        })
+    }
+    //  item이 F이면 active 상태로 처음 작성시 item == 'F' 일때의 localStorage 저장 역할
+    else
+        localStorage.setItem(value,'F');
+
+    //버튼 클릭시 가장가까운 li태그 삭제 및, 해야할 count 값 줄이기
+    document.querySelectorAll('.destroy').forEach($el => $el.addEventListener('click', function (event) {
+        event.target.closest('li').remove();
+        document.querySelector('.todo-count > strong').innerText = --$count;
+        localStorage.removeItem(event.target.closest('div').children[1].innerText);
+        getUserIdAndDeleteTodolist(document.querySelector('#user-list > .active').innerHTML, $el.parentNode.children[1].innerText);
+    }))
+
+    // 체크박스 클릭시 li태그에 class속성 추가 및 text에 중간작대기 생성 (클릭 취소하면 class속성 completed 추가 및 text원래대로)
+    // 체크박스 클릭 혹은 클릭 취소 시  localStorage 속성도 변화시킨다.
+    document.querySelectorAll('.toggle').forEach($el => $el.addEventListener('click', function (event) {
```

Comment:
```suggestion
    document.querySelectorAll('.toggle').forEach($el => $el.addEventListener('click', function ({ target }) {
```

`event.target`에 많은 접근을 하고 계신데요.
이렇게 사용하면 조금이나마 코드를 간결하게 표현할 수 있습니다.
이를 [**구조 분해 할당**](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)이라고 표현합니다! 😄